spec/features/user/authentication/unauthentic_user_cannot_view_another_users_data_spec.rb

```diff
@@ -2,7 +2,9 @@
 
 RSpec.feature "Unauthenticated users security" do
   before(:each) do
-    setup
```

Comment:
Eliminating the use of SlowHelper and methods makes for less confusing test suite. Nice!